Dockerfile

```diff
@@ -120,8 +120,10 @@ COPY . /ash/
 #
 # Install CDK Nag stub dependencies
 #
+# Update NPM to latest
+RUN npm install -g npm
```

Comment:
Does this need to be a separate `RUN` step (resulting in an additional layer being created)?

Could the `npm install -g npm` be the first command run before the `cd ... && ...` commands in the next  `RUN` step - similar to the pattern for `yum update && yum upgrade -y` pattern in many `Dockerfile`s?

Reply:
@climbertjh2 we can add it on the same run call, I'll update that now. Not a bad suggestion